Xamarin.Forms.Core/ResourceDictionary.cs

```diff
@@ -327,6 +327,13 @@ void OnValuesChanged(params KeyValuePair<string, object>[] values)
 			ValuesChanged?.Invoke(this, new ResourcesChangedEventArgs(values));
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void Reload()
+		{
+			foreach (var mr in MergedResources)
+				OnValuesChanged(mr);
```

Comment:
I'd like to avoid doing that

Reply:
In the latest changes it has at least changed to internal. I know it's not pretty and open to alternatives. But the problem is that the actual value doesn't really change. Depending on the theme the implicit operator will return a different color and that is why it's not picked up normally.

The only time this happens is when the theme changes on OS level and then a number of expensive operations already happen to redraw the whole screen already.